test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/LuaE2E.java

```diff
@@ -92,14 +95,21 @@
     public void setUp() throws Exception {
         queryClient(swWebappHostPort);
 
-        trafficController(entryProvider, "/nginx/entry/info?backend=" + nginxHostPort.host() + ":" + nginxHostPort.port());
```

Comment:
Remove the declaration of `nginxHostPort` (line#89) if you don't need it anymore